boards/arm/b_l072z_lrwan1/doc/b_l072z_lrwan1.rst

```diff
@@ -0,0 +1,225 @@
+.. _b_l072z_lrwan1_board:
+
+ST B-L072Z-LRWAN1 Discovery kit
+###############################
+
+Overview
+********
+
+This Discovery kit features an all-in-one open module CMWX1ZZABZ-091 (by Murata).
+The module is powered by an STM32L072CZ and an SX1276 transceiver.
+
+This kit provides:
+
+- CMWX1ZZABZ-091 LoRa |reg| /Sigfox |trade| module (Murata)
+
+        - Embedded ultra-low-power STM32L072CZ Series MCUs, based on Arm |reg| Cortex |reg| -M0+ core, with 192 Kbytes of Flash memory, 20 Kbytes of RAM, 20 Kbytes of EEPROM
```

Comment:
Try to keep lines <80 chars long:

    - Embedded ultra-low-power STM32L072CZ Series MCUs,
      based on Arm* Cortex* -M0+ core, with 192KB
      of Flash memory, 20KB of RAM, 20KB of EEPROM